src/main/java/com/squareup/javawriter/CodeWriter.java

```diff
@@ -214,6 +215,9 @@ private void emitLiteral(Object o) {
     if (o instanceof TypeSpec) {
       TypeSpec typeSpec = (TypeSpec) o;
       typeSpec.emit(this, null);
+    } else if (o instanceof AnnotationSpec) {
+      AnnotationSpec annotationSpec = (AnnotationSpec) o;
+      annotationSpec.emit(this, true);
```

Comment:
Agreed. I think the best behavior here is to compute it magically, possibly by measuring how long the lines are. I'll keep this as-is for now, but I want to do better. (Later).